From c391bd5c645398f2978a9dc2755fd076a97ea340 Mon Sep 17 00:00:00 2001
From: Jakub Steiner <jimmac@gmail.com>
Date: Tue, 14 Jan 2014 17:00:23 +0100
Subject: [PATCH] classic: shade panel in overview

- rather than using the top bar styling (negative space),
  base the overview panel on the classic grey and "darken"
  for overview
---
 data/gnome-classic.css | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/data/gnome-classic.css b/data/gnome-classic.css
index ee4ec52..29a4132 100644
--- a/data/gnome-classic.css
+++ b/data/gnome-classic.css
@@ -1865,12 +1865,10 @@ StScrollBar {
   border-bottom: 1px solid #666;
   app-icon-bottom-clip: 0px; }
   #panel:overview {
-    background-color: #000;
-    background-gradient-end: #000;
-    border-top-color: #000;
-    border-bottom: 1px solid #000; }
+    background-color: #e0e0e0;
+    background-gradient-end: #d4d4d4; }
     #panel:overview .panel-button {
-      color: #fff; }
+      color: #222728; }
   #panel .panel-button {
     -natural-hpadding: 8px;
     -minimum-hpadding: 4px;
-- 
2.12.0

